aacparser.go

```diff
@@ -837,8 +837,8 @@ func (adts *ADTS) adts_fixed_header() error {
 		adts.Profile, _ = adts.reader.ReadBitsAsUInt8(2)        // profile object
 		adts.Profile += uint8(1)                                // profile object
 		adts.sfi, _ = adts.reader.ReadBitsAsUInt8(4)            // sampling_frequency_index
-		if adts.sfi > 12 {
-			return fmt.Errorf("Scale Factor Index (%d) out of acceptable range (0-12)", adts.sfi)
+		if adts.sfi > 11 {
```

Comment:
ISO/IEC-14496-3 table 1.18 specifies valid values from 0x00-0x0C.  Unfortunately this isn't the right place to fix this.